webpack/assets/javascripts/react_app/components/Bookmarks/components/BookmarkForm/BookmarkForm.js

```diff
@@ -41,6 +40,7 @@ const BookmarkForm = ({
       initialValues={initialValues}
       validationSchema={bookmarkFormSchema}
       onCancel={onCancel}
+      onSuccess={setModalClosed}
```

Comment:
can we use the modal hook and call it inside ForemanForm?

Reply:
I'd prefer to refactor the modal here in a separate PR